packages/flags/src/flags/buildInfo.ts

```diff
@@ -28,7 +28,9 @@ export async function getBuildInfo(): Promise<BuildInfoFile> {
         }
     }
 }
-export let env: BuildInfoFile
+export let env: BuildInfoFile = {
+    channel: 'beta',
```

Comment:
```suggestion
    channel: 'stable',
```
?

and access to this variable before call setupBuildInfo should be a fatal error.